xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -184,21 +198,13 @@ bool CRendererBase::Configure(const VideoPicture& picture, float fps, unsigned o
   m_lastHdr10 = {};
   m_HdrType = HDR_TYPE::HDR_NONE_SDR;
   m_useHLGtoPQ = false;
+  HDR_STATUS hdrStatus = CWIN32Util::GetWindowsHDRStatus();
+  m_isHdrEnabled = (hdrStatus == HDR_STATUS::HDR_ON);
```

Comment:
Maybe rename variable to reflect is "initial HDR condition"

`m_initiallyHDRenabled` ?